examples/ios/objc/RealmExamples.xcodeproj/project.pbxproj

```diff
@@ -934,6 +1065,20 @@
 			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
 			showEnvVarsInLog = 0;
 		};
+		F18DC0A41DC2B197007D2D69 /* Get Device IP Address */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "Get Device IP Address";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "LOCAL_IP_ADDR=$(ipconfig getifaddr $(netstat -f inet -rant | awk '/default/ { print $6; exit }'))\necho \"static NSString * const kIPAddress = @\\\"$LOCAL_IP_ADDR\\\";\" > \"$SRCROOT/Draw/Constants.h\"";
```

Comment:
If you replace this with the same script as [RealmTasks](https://github.com/realm/RealmTasks/blob/83cd2dfc3c72a8f1297b4f23609e20df6ed014f3/RealmTasks%20Apple/RealmTasks.xcodeproj/project.pbxproj#L640), you won't need the compile-time conditional in the code.